docs/psmdb-pro.md

```diff
@@ -0,0 +1,23 @@
+# Percona Server for MongoDB Pro
+
+Percona Server for MongoDB Pro includes [solutions](#solutions) typically demanded by large enterprises. Percona Server for MongoDB Pro is wrapped in packages created and tested by Percona. These packages are supported only for paying Percona Customers with a subscription.
+
+[Become a Percona Customer](https://www.percona.com/about/contact){.md-button}
+
+## Solutions
+
+Find the list of solutions available in Percona Server for MongoDB Pro:
+
+| Name                                | Version added | Description  | 
+| ----------------------------------- | ------------- | -------------
+| [FIPS compliance module](fips.md)| [6.0.9-7](release_notes/6.0.9-7.md) | Enables US organizations to introduce FIPS-compliant encryption and thus meet the requirements towards data security |
```

Comment:
```suggestion
| [FIPS support ](fips.md)| [6.0.9-7](release_notes/6.0.9-7.md) | Enables US organizations to introduce FIPS-compliant encryption and thus meet the requirements towards data security |
```
I wouldn't say we have added any module, it's rather FIPS support when PSMDB can use proper SSL/TLS libraries